allure-commandline/src/test/java/io/qameta/allure/CommandLineTest.java

```diff
@@ -224,4 +224,15 @@ public void shouldPrintPluginList() throws Exception {
         assertThat(run)
                 .isEqualTo(NO_ERROR);
     }
+
+    @Test
+    public void shouldHandleUnknownOptions() {
```

Comment:
`-q` is not an unknown option by the way. See https://github.com/allure-framework/allure2/blob/0d1517e4d44d30812ea38b75d82ac27f7343712c/allure-commandline/src/main/java/io/qameta/allure/option/VerboseOptions.java

Unknown options works fine:

```bash
$ allure --unknown

Could not parse arguments: Expected a command, got --unknown
Usage: allure [options] [command] [command options]
  Options:
    --help
      Print commandline help.
    -q, --quiet
      Switch on the quiet mode.
      Default: false
    -v, --verbose
      Switch on the verbose mode.
      Default: false
    --version
      Print commandline version.
      Default: false
```